client/me/purchases/purchases-list/index.jsx

```diff
@@ -96,7 +97,7 @@ class PurchasesList extends Component {
 	renderMembershipSubscriptions() {
 		const { subscriptions } = this.props;
 
-		if ( ! subscriptions.length ) {
+		if ( ! subscriptions.length || this.isDataLoading() ) {
```

Comment:
Do you remember why this was added?

Reply:
yes, the memberships subscriptions were loading before the wordpress.com purchases. This extra condition made sure they showed up at the same time.